lib/plugin.go

```diff
@@ -27,14 +31,17 @@ type Plugin struct {
 // func(req lib.Request, res *lib.Response) (err error)
 // see [examples/plugin]() for more info
 func (p *Plugin) Do(ctx context.Context, conductor string, rcvr interface{}) (err error) {
+	if p.Log == nil {
+		p.Log = log.Std
```

Comment:
I decided it would be more expected to use the std logger rather than `github.com/go-pkgz/lgr`